Version 0.0.4 Fixed MQTT and Added Calibration

brickmen opened the pull request:
The ESP8266 now works with MQTT and allows for linear calibration.

Other fixes:
- Disabled SoftAP when STA is working

- Increased JSON buffer size

- Added MQTT debug info to serial

NOTE: This will break the travis.yml file which will need to be reset with original version before merge

beLizzard1 committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into MQTT-Integration

Commit:
added versioncode to web interface

brickmen committed:
modified to work with forked repo

changed repo from rjames to brickmen

brickmen committed:
Update .travis.yml

brickmen committed:
Updated the Secure API key

brickmen committed:
Remade API key and re-inserted it

brickmen committed:
Brought in new key from mqtt branch manually

Commit:
travis setup local

brickmen committed:
local modified version for travis

beLizzard1 committed:
Added some comments

Cleaned up some of the excess/commented out code. Added a few comments that caught my eye that might need explaining later on.

beLizzard1 committed:
Oops

Missed a semicolon... Serves me right for not actually paying attention.

beLizzard1 committed:
Fixed another semicolon mistake

Can you tell that I'm exhausted.

brickmen committed:
Corrected use of '

Accidentally used " instead of '

brickmen committed:
Update .travis.yml

Commit:
Merge remote-tracking branch 'upstream/MQTT-Integration'
Adding the changes made by Rob

Commit:
Merge branch 'master' of https://github.com/brickmen/WiFiSensor

Commit:
Re-uploaded working Travis file

Commit:
Added Serial message for successful publication to MQTT

Commit:
Silly mistake with characters and strings, try again

Commit:
Added Failed MQTT Serial Message

Commit:
Changes to the Serial output to ensure it fits on one line

Commit:
Made the MQTT error serial output very verbose

Commit:
Found issue int the code theat the MQTT topic names were not set.
Corrected this and uploaded.

Commit:
More Verbose MQTT Errors
Added even more the the MQTT error message on serial

Commit:
bRought in code for MQTT Config
Introduced code from m

Commit:
Code from mqtt.ino into main
To setup mqtt username and password i have introduced this into the main
code

Commit:
Changed the MQTT connect results to be more useful
Also edited this in arduino ide some error aboult CRLF was noted

Commit:
Moved MQTT publish to

Commit:
moved mqtt code and debug to mqtt.ino

Commit:
fix some brackets

Commit:
Silly Typo fix

Commit:
replaced print with println in mqtt.ino

Commit:
Casting msg to string for println

Commit:
Trying with every line separated

brickmen committed:
Overriding the MQTT logic in Load Config

To Test the systen the logic is overridden for now

brickmen committed:
Modified Config File

Added calibration and un-nested mqtt settings

brickmen committed:
Modified Confif File 2

fixed as<float> casting

brickmen committed:
Increased jsom buffer of mqtt settings

brickmen committed:
Updated ReadMe ready to Pull

Tested and working ready to send back to Rob

brickmen committed:
Applied a function to deepsleep and DHT check

Untested changes to add deepsleep functionality (do NOT use as it diables any way to config device)

brickmen committed:
Modified deepsleep function

A setup has been configured that may work for deep sleep need to test

brickmen committed:
Issue with timer defined in Loop, 

moved it out of if loop, shouldn't really matter but lets see

brickmen committed:
Missed a semi-colon

brickmen committed:
Removed if timer in deepsleep

brickmen committed:
Merge branch 'deepsleep'

Commit:
Added WebServer Override to DeepSleep

Untested

Commit:
Typo Fix

Fixed some typos

Commit:
Another Typo

Commit:
Another typo again

Commit:
Added a delay in ds loop

Commit:
Set 10s sleep interval

Commit:
Added MQTT retry method

Commit:
Fixed Typos

Commit:
Another typo

Commit:
Get there eventually

Commit:
Increased sleep time to 10 mins

Commit:
Added timing control to DeepSleep and Normal

Commit:
Forgot to increment version no

beLizzard1 reviewed (changes requested):
Don't commit anything relevant to the build process to the repository.

Commit:
Untested v0.07

Added the configuration needed for dual DHT and LED control

 "board-conf": {
    "led-grn-d1": true,
    "led-red-d2": true,
    "dht-1-d5": true,
    "dht-2-d6": false,
    "batt-1-a0": false
  },

Commit:
some fixes for v0.0.7

Not all the fixes have been setup, need to sort analog read and investigate Vcc

Commit:
Untested Fix 1 v0.0.7

Commit:
Fix 2 0.0.7

Commit:
Fix 3 0.0.7

Sorted an issue with the JSON buffer and added offsets for dht2

Commit:
Fix 4 0.0.7

Added config check function

Commit:
Fix 4.1 0.07

Added a println to identify where the crash happens

Commit:
Fix 6 0.0.7

Commit:
Fix 7 0.0.7

Removed the voltage corrector to establish the adjustment needed.

Commit:
v0.0.8

Added unified MQTT publish to both DS and normal mode. Set Soft AP to close after an MQTT update.

Commit:
v0.0.8 fix 1

typos

Commit:
v0.0.9 

Added voltage multiplier to code

Commit:
v0.0.9 fix1

Added a dht reading when in dsoverride

Commit:
v0.0.9 fix2

unusual bug where travis cannot "see" the other functions attempting a fix

Commit:
v0.0.9 fix 3

sorted some issues witht he display of °C

Commit:
v0.0.9 fix 4

formatting fix...weird error still happening

Commit:
v0.0.9 fix 5

typo error

Commit:
v0.0.9 fix 6

remove voltage multiplier check as it is not needed and increased buffer size of the calibration json

brickmen committed:
v0.1.0

Minor fixes, added version no to serial as well as some additional info. Double checked the LED settings.

brickmen committed:
v0.1.0 fix 1

Added print to serial in dht measure within deep sleep override

brickmen committed:
v0.1.0 fix 2

Fixed a minor issue with ds override counting

brickmen committed:
v0.1.0 fix 3 

minor changes to sort the LED flashing and the serial output in deep sleep
